contrib/ci/fedora.sh

```diff
@@ -36,3 +36,5 @@ rpmbuild -ba "${QUBES_MACRO[@]}" build/fwupd.spec
 
 mkdir -p dist
 cp $HOME/rpmbuild/RPMS/*/*.rpm dist
+
+modprobe netdevsim
```

Comment:
I think you need to do this before we build the package -- i.e. move this up the file a bit.